lib/people.ts

```diff
@@ -10,25 +14,37 @@ export type Person = {
   bio: string;
   profile_photo: string | null;
 };
-
+/**
+ * @typedef PartialPerson
```

Comment:
Just to make sure, we have this b/c it would be more costly to fetch people' bios if we don't need them right? We don't actually use a lot of the elements of partial_person on the project/sig pages

Reply:
yeah, I believe that's right. less data to transfer esp when it's not needed. the photos aren't currently used but I included them incase we wanted to have mini-thumbnails for each person on a proj or in a sig.